AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/internal/core/handler/TraceHandler.java

```diff
@@ -98,7 +99,7 @@ public void process(Element element, EComponentHolder holder) throws Exception {
 
 		logEnterInvoke.arg(getEnterMessage(method, executableElement));
 		thenBody.add(logEnterInvoke);
-		JVar startDeclaration = thenBody.decl(getCodeModel().LONG, "start", currentTimeInvoke);
+		JVar startDeclaration = thenBody.decl(getCodeModel().LONG, "traceStart_", currentTimeInvoke);
```

Comment:
I think we should use `GENERATION_SUFFIX` instead of hardcoding the underscore into every string.